www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -78,6 +78,31 @@ function (declare, domClass, dom, style, fx, Toggler, on, query, Memory,
     return filepath.substr(0, endPos);
   }
 
+  /**
+   * Creates a bug step enumeration icon.
+   * @param value {integer} - Content of the enumeration icon.
+   * @param type {error|fixit|info} - Class type of the enumeration icon.
+   */
+  function createBugStepEnumeration(value, type) {
+    return dom.create('span', {
+      class : 'checker-enum ' + type,
+      innerHTML : value
+    });
+  }
+
+  /**
+   * Parsing the bug step message and replace the `(fixit)` string with an icon.
+   */
+  function parseBugStepMsg(message) {
+    var ret = message;
+
+    if (message.indexOf('(fixit)') > -1)
+      ret = '<span class="tag fixit">fixit</span>'
+        + message.replace('(fixit)', '');
```

Comment:
The text is appended as ` (fixit)` with a space before the tag. While this is a very minor stuff, I think we should stript in entirely.